Firestore/src/V1/BatchGetDocumentsResponse.php

```diff
@@ -211,4 +211,3 @@ public function getResult()
     }
 
 }
-
```

Comment:
We should not make changes to generated files - they'll just get changed back. If you want to add newlines to the end of protobuf files, that change should be made to protobuf or to our post-processor. Personally i don't find it important enough to expend that effort, but feel free to make the change if you disagree!

But for simplicity and noise reduction, we shouldn't push manual changes to generated files.